udata/tests/dataset/test_dataset_events.py

```diff
@@ -104,7 +104,7 @@ def test_publish_message_resource_removed(self, mock_req):
             dataset.remove_resource(resource)
 
         mock_req.assert_called_with(
-            f"{current_app.config['RESOURCES_ANALYSER_URI']}/api/resource/deleted/",
+            f"{current_app.config['RESOURCES_ANALYSER_URI']}/api/resources/",
```

Comment:
For information, it's not part of this issue but at some point I'd like to make the _routes_ really REST, meaning on this call it would be a `DELETE` on `/api/resources/{resource_id}`, with no payload.

Does that make sense to you?

Either we prepare it in this PR, either in another one, as you wish.